CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## 1.6.3 (2019-07-23)
+
+-   Improved parsing for some new cases such as:
```

Comment:
Unclear. New in YML? Newly parsed for the extension? But then can we call it “improvement” if it didn’t exist to begin with? I’d say “keep it simple”.

```suggestion
-   Improved parsing for:
```